CIP-XXXX/README.md

```diff
@@ -0,0 +1,60 @@
+---
+CIP: ?
+Title: CIP-30 ext: Bulk transaction signing
+Category: Wallets
+Status: Proposed
+Authors:
+    - Martín Schere
+    - Ola Ahlman <ola.ahlman@tastenkunst.io>
+Implementors: 
+    - JPG Store
+    - Eternl
+    - Typhon
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/443
+Created: 2023-09-03
+License: CC-BY-4.0
+---
+
+## Abstract
+This CIP extends [CIP-30 (Cardano dApp-Wallet Web Bridge)](https://cips.cardano.org/cips/cip30/) to provide an additional endpoint for dApp to sign multiple transactions in bulk.
+
+## Motivation: why is this CIP necessary?
+Currently, there is no way to sign multiple transactions in bulk, and the experience of signing a chain of transactions is suboptimal. We propose the addition of a signTxs endpoint that enable wallets to create an array of interconnected transactions and sign them all at once.
+
+## Specification
+
+### TransactionSignatureRequest
+
+```
+type = TransactionSignatureRequest {|
+  cbor: cbor\<transaction>,
+  partialSign: bool = false,
+|};
+```
+
+Used to represent a single transaction awaiting a user's signature. More details on {partialSign} can be found in [api.signTx](https://cips.cardano.org/cips/cip30/#apisigntxtxcbortransactionpartialsignboolfalsepromisecbortransactionwitnessset) defined in CIP-30.
+
+### api.cip????.signTxs(txs: TransactionSignatureRequest[]): Promise\<cbor\<transaction_witness_set>[]>
+
+Errors: `APIError`, `TxSignError`
+
+Signs a chain of transactions, which can be described as a sequence of interconnected transactions where each subsequent transaction depends on the previous one. The returned array values of the witness set directly correspond to the elements in the `txs` parameter, aligning the witness set at index 0 with the transaction at index 0, and so forth.
+
+## Rationale: how does this CIP achieve its goals?
+Allowing for bulk signing of transactions can greatly increase user experience in certain situations. There are however certain things that should be considered by wallets implementing this CIP, namely user visibility of what is signed. Though not explicitly specified in this CIP, as it would be up to the wallet to find a good solution, the wallet should make it clear to the user that multiple transactions are to be signed, and to give a clear overview of what is signed. In addition to visibility, the wallet shall process input transaction array in order to allow transactions to be chained by accepting previous transaction in the array to be used as input in a following.
+
+## Path to Active
+
+### Acceptance Criteria
```

Comment:
Some questions about this section:

- for demonstration I added a checkbox format here (https://github.com/cardano-foundation/CIPs/pull/587/commits/681092ed2abb5bb6ee0c3a3722b1c36b5efdaa18)... if your single _Implementation_ item is done, please confirm this & it would also match similar proposals if you would add a checkbox in front of it in the same style (especially helpful if this item is _not_ done yet).

- It sounds like your item might be done because of the term "Already".  Therefore I would have to guess that you have a single implementation for both wallets & dApps but that it's not considered `Active` until you have two for each.  Can you clarify the language here to explicitly indicate whether that's true?  The suffix `(s)` on both these terms leaves it unclear.

Reply:
The `Path to Active` section was a bit confusing to me. Who decides the criteria(s) for when a CIP should be considered accepted, the proposer or the CIP editors? If proposer, are there any guidelines for what would be acceptable criteria?

As I wasn't sure what to put here I just added the current state basically. I know Typhon has it enabled already directly in the main api as `api.signTxs()` while for now, as it wasn't clear at the time for draft namespacing, Eternl has it added as `api.experimental.signTxs()`. JPG store which initiated the first PR, now closed, is the only DApp so far to my knowledge that utilizes it but understandable as it hasn't been defined and standardized yet.

Reply:
I now marked checkboxes as completed and pushed. But I'm uncertain about how to re-word this entire section. I would argue that PR is ready to become active as it's already today implemented by two wallets and one dapp. If that is considered enough by CIP editors that is.

Reply:
@Scitz0 the only time CIP editors would decide the acceptance criteria is when a CIP author hasn't written anything in the CIP on that subject (or the CIP was adopted before that requirement was imposed).  Generally the proposer writes that section & the CIP editors follow a consensus about whether it's a reasonable definition.  As I suspected, your CIP might be considered `Active`.

The question of whether experimental implementations contribute to `Active` status is something I don't think we've come across before, so as far as I know we don't have a precedent about it.  I could answer in theory but I don't work directly on these implementations so I don't know what would work best for the community of implementors, so we should get answers from editors
- @Ryun1 who's been working on standardising the wallet API for some time now
- @KtorZ who formulated this definition of `Path to Active` and who we've been relying up on settle questions of definition and to set these precedents when unclear or disputed... he's been away for a while with no definite date posted for return but we are all hoping that will be soon.

Reply:
Moving this to `proposed` is reasonable until the existing implementers switch over to this API as described, so switch to using the namespaced `.cipxxxx.signTxs()`.